### PR TITLE
"Gruppe" anstatt "Arbeitsbereich"

### PR DESCRIPTION
**Pull request by acka47**
Adjustment due to our new organizational status

### CODE REVIEW

File: team.json [hunk 1]
```diff
@@ -86,7 +86,7 @@
     "en": "The lobid team"
   },
   "description": {
-    "de": "lobid wird implementiert und betrieben vom <a href=\"https://www.hbz-nrw.de/produkte/linked-open-data\">Arbeitsbereich Linked Open Data des hbz</a>",
+    "de": "lobid wird implementiert und betrieben von der Gruppe \"Offene Infrastruktur\" des hbz",
     "en": "lobid is implemented and maintained by <a href=\"https://www.hbz-nrw.de/produkte/linked-open-data\">hbz's Linked Open Data group</a>"
     },
```
Comment (dr0i) [inline team.json]: both languages should reflect the same informations

**Comment by acka47** [comment]
Done. Please review again. (Unfortunately, we don't have an English HTML version anymore...)

**Comment by dr0i** [comment]
Deployed, see https://lobid.org/team/. Closed.

File: team.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
   },
   "description": {
     "de": "lobid wird implementiert und betrieben von der Gruppe \"Offene Infrastruktur\" des hbz",
-    "en": "lobid is implemented and maintained by <a href=\"https://www.hbz-nrw.de/produkte/linked-open-data\">hbz's Linked Open Data group</a>"
+    "en": "lobid is implemented and maintained by hbz's \"Open Infrastructure\" group"</a>"
     },
```
Comment (dr0i) [inline team.json]: uh - this actually breaks the json structure. Fixed in master.

**Pull request merged**